### PR TITLE
Created a block to switch between integer signals

### LINKED ISSUE #1471: Chiller staging: capacities
Implement the following chiller staging sequences:

- [x] Capacity requirement.
- [x] Design and minimal capacities.

The outputs of these sequences are inputs into the calculation of operating and staging part load ratios.

### LINKED ISSUE #1473: Create a switch between integer inputs
Analogous to `Buildings.Controls.OBC.CDL.Logical.LogicalSwitch` create a `Buildings.Controls.OBC.CDL.Logical.IntegerSwitch`.

### PR DESCRIPTION
**Pull request by milicag**
CLoses #1473

### CODE REVIEW

File: Buildings/Controls/OBC/CDL/Logical/IntegerSwitch.mo [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
 
 equation
   y = if u2 then u1 else u3;
+
   annotation (
 defaultComponentName="intSwi",
 Documentation(info="<html>
 <p>
-A block to switch between two integer signals.
+Block that switches between two integer signals and produces one of it at its output.
```
[the remark this answers is not in the document]
Comment (mwetter) [reply Buildings/Controls/OBC/CDL/Logical/IntegerSwitch.mo]: Thanks, I changed the comment.

[the remark this answers is not in the document]
Comment (JayHuLBL) [reply Buildings/Controls/OBC/CDL/Logical/IntegerSwitch.mo]: @mwetter there is a typo `... and produces one of it at its output.`

[the remark this answers is not in the document]
Comment (mwetter) [reply Buildings/Controls/OBC/CDL/Logical/IntegerSwitch.mo]: That looks correct to me.

[the remark this answers is not in the document]
Comment (milicag) [reply Buildings/Controls/OBC/CDL/Logical/IntegerSwitch.mo]: Too much logic :)

[the remark this answers is not in the document]
Comment (JayHuLBL) [reply Buildings/Controls/OBC/CDL/Logical/IntegerSwitch.mo]: Thought it should be `...as its output`. :)

[the remark this answers is not in the document]
Comment (milicag) [reply Buildings/Controls/OBC/CDL/Logical/IntegerSwitch.mo]: It's already changed, please see the code.

**Pull request merged**